rate_counter/rate/tasks.py

```diff
@@ -0,0 +1,102 @@
+# -*- encoding: utf-8 -*-
+from __future__ import absolute_import
+from celery.task import periodic_task
+from .models import Institute, StudentGroup, Subject
+import requests
+from bs4 import BeautifulSoup
+import re
+
+
+@periodic_task(run_every=60*60)  # період запуску в секундах (кожну годину)
+def get_institutes_id():
+
+    url = 'http://www.lp.edu.ua/node/40?inst=1&&semestr=1&semest_part=1'
+    r = requests.get(url)
+    soup = BeautifulSoup(r.content, "html.parser")
+
+    # я подивився структуру сторінки розкладу і виявив, що всі назви і значення інститутів і груп
+    # завжди містяться між тегами <option>. Але також там містяться значення поточного семестру і його половини,
+    # тобто, весняний(0), осінній(1), перша(1) і друга(2).
+    # Список всіх інститутів міститься на будь-якій сторінці розкладу,
+    # а список груп, тільки на сторінці де вказаний інститут до якого вони належать.
+
+    # Заповнення бази даних інститутами
+    for option in soup.find_all('option'):
+        if len(option.text) == 4:
+            print option.text
+            if not Institute.objects.filter(name=option.text).exists():
```

Comment:
Instead of this check, I guess You can use Django [`get_or_create`](https://docs.djangoproject.com/en/dev/ref/models/querysets/#get-or-create). It will make it cleaner. check in docs it is pretty straightforward